docs/prd/aws-assume-root-identity.md

```
@@ -0,0 +1,714 @@
+
+**Validation**:
+- `target_principal` must be a valid 12-digit AWS account ID
+- `task_policy_arn` must match pattern `arn:aws:iam::aws:policy/root-task/*`
+- `duration` must be between 1 and 900 seconds
+
+### FR-3: AWS-Managed Task Policies
+
+**Requirement**: Support all AWS-managed root task policies
+
+**Supported Task Policies**:
+| Task Policy ARN | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)